bin.src/gen3.py

```diff
@@ -30,7 +30,7 @@
 def main():
     registry = gen3.getRegistry()
     datastore = gen3.getDatastore(registry)
-    if not registry.findDataUnitEntry("Instrument", {"instrument": "HSC"}):
+    if not registry.findDimensionEntry("Instrument", {"instrument": "HSC"}):
         gen3.registerInstrument(registry)
```

Comment:
More for myself: the assumption is that `registerInstrument` (and similar methods for `PhysicalFilter` and `Detector`) will be moved to `daf_butler` when gen3 is ready for more than just HSC, correct?

Reply:
This `registerInstrument` just forwards to a `HyperSuprimeCam.register` method in obs_subaru, which in turn actually registers all of the `PhysicalFilters` and `Detectors`.  I don't have any immediate plans to move that out of obs_subaru (and I'd like to keep any knowledge of specific cameras out of daf_butler), but once we've implemented this in other obs packages I could imagine us refactoring a bit to remove code duplication.